src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClientFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Options;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextImplementations
+{
+    /// <summary>
+    ///     Factory class to create Durable Client to start works outside an azure function context.
+    /// </summary>
+    public class DurableClientFactory : IDurableClientFactory
+    {
+        private readonly DurableTaskExtension durableTaskConfig;
```

Comment:
Pulling in `DurableTaskExtension` adds in a whole bevy of things that probably aren't needed to create clients (including starting a `TaskHubWorker`, which will start actively pulling orchestration and activity items off of the task hub it is listening for).

Instead, consider refactoring `DurableClient` to take only the things it needs from `DurableTaskExtension` (which is largely just the `GetClient()` method).